runtime/lua/vim/treesitter/highlighter.lua

```diff
@@ -45,9 +46,10 @@ end
 
 --- Creates a new highlighter using @param tree
 ---
----@param tree The language tree to use for highlighting
----@param opts Table used to configure the highlighter
----           - queries: Table to overwrite queries used by the highlighter
+---@param tree LanguageTree Parser object to use for highlighting, see |LanguageTree|
```

Comment:
I'd rather have (see other comments):
```suggestion
---@param tree LanguageTree |LanguageTree| object to use for highlighting.
```

Reply:
_That_ I can (and will) do! 👍